### PR TITLE
Add case study for dead code detection

### PR DESCRIPTION
**Pull request by Agares**

### CODE REVIEW

**Review by darach — APPROVED** [review]
LGTM 🚀  This is great to get this production use case documented just in time for the wayfair case study rollout. Superstar!

File: blog/2021-11-09-php-dead-code-detection.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Dead Code Detection in PHP
```
Comment (Licenser) [inline blog/2021-11-09-php-dead-code-detection.md:1]: we will need a header for this along the lines of the other blogs :) you can copy & paste and just adjust the name / tags / time. I think `casestudy` is a good tag to add for this

**Review by Licenser — CHANGES_REQUESTED** [review]

File: blog/2021-11-09-php-dead-code-detection.md (hunk 1 above)
Comment (darach) [reply blog/2021-11-09-php-dead-code-detection.md:1]: Yep. I'll add a header on the case study branch

Comment (darach) [reply blog/2021-11-09-php-dead-code-detection.md:1]: We should add this to the top level case study too - so that needs an extra paragraph

**Comment by Licenser** [comment]
This wasn't ready yet to be merged the header is not in which means there is a bunch of things missing

**Comment by Agares** [comment]
No worries, it only got merged into Darach's branch

**Comment by Licenser** [comment]
makes more sense now :) missed taht

**Pull request merged**